common/pinot/pinot_client.go

```diff
@@ -70,6 +70,18 @@ func (c *PinotClient) Search(request *SearchRequest) (*SearchResponse, error) {
 	return c.getInternalListWorkflowExecutionsResponse(resp, request.Filter, token, request.ListRequest.PageSize, request.MaxResultWindow)
 }
 
+func (c *PinotClient) SearchAggr(request *SearchRequest) (AggrResponse, error) {
```

Comment:
this method is missing tests as far as I see

Reply:
Added